sdk/azidentity/client_certificate_credential.go

```diff
@@ -96,7 +98,7 @@ func NewClientCertificateCredential(tenantID string, clientID string, certificat
 	if err != nil {
 		return nil, err
 	}
-	c, err := newAADIdentityClient(authorityHost, pipelineOptions{HTTPClient: options.HTTPClient, Retry: options.Retry, Telemetry: options.Telemetry})
```

Comment:
The defaults should be adding after line 41 in DefaultClientCertificateCredentialOptions()

Reply:
Fixed